pomodoro@gregfreeman.org/README.md

```diff
@@ -1,4 +1,69 @@
+
+**Auto Start Options**
+- **Purpose**: Auto start settings allow for an uninterrupted workflow by automatically starting the next session—be it a Pomodoro, short break, or long break—without user interaction.
+- **Usage**: 
+  - **Pomodoro and Short Break**: If auto start is enabled for these timers, the next session will commence immediately after the current one ends, without showing any dialog, regardless if the dialog window option is enabled or disabled
+  - **Long Break**: The auto start for a long break can work concurrently with dialog windows. When both are enabled, a dialog window will appear at the start of the long break showing a countdown. If the auto start after long break is also enabled, this dialog will automatically close when the countdown finishes, and a new Pomodoro will begin.
+
+### Best Practices for Configuring Settings
+
+- **Maximize Focus**: Enable auto start for Pomodoros and short breaks if you want to maintain a strict work rhythm without interruptions, ideal for deep focus sessions where maintaining momentum is crucial.
+- **Stay Informed and Rested**: Use dialog windows with long breaks to stay informed about the break period but still enjoy the benefit of automatic transition back to work, ensuring you don’t skip necessary rest.
+- **Balance Flexibility and Structure**: If your work requires flexibility, use dialog windows for Pomodoros and short breaks and disable auto start to decide when you're truly ready to move on. This setup helps manage work that may need sudden shifts or extensions in activity time.
+
+### Tailoring to Your Workflow
+
+Experiment with different configurations to find the optimal setup that enhances your productivity without causing burnout. The goal is to leverage the Pomodoro technique in a way that best suits your daily routines and work demands, adjusting settings as needed to accommodate changes in your work intensity or focus requirements.
+
+## Features
+* Customize timer durations for the pomodoro, short and long breaks
+* See how many pomodoros you have completed
+* Configure what is shown in the panel, options include an icon and the timer counting down
+* Optionally play and configure sounds such as timer start and end
+* Show dialog windows at the end of the timers so you know to take a break
+* Auto start the timer when you first log in
+* Set a keyboard shortcut to open the applet menu and then use arrow keys to select options
+* Quickly reset the timer state or skip to the next timer if you want to end early
+
+## Changelog
+- 1.0.1
+    - Added option to automatically start the timer on log in
+- 1.0.0
+    - Added keyboard shortcut to open the applet
+    - Added new dialog option to show after a pomodoro ends
+    - Added applet menu option to skip the current timer, useful if you want to end early
+    - Updated code to use ES6 for easier updates in future
```

Comment:
Just a note that if you put the Changelog into a separate CHANGELOG.md file, the Spices website will render it separately from the README for people who are interested. (Take a look at other Spices that do this if you want an example and you decide that this is something you want.)